### PR TITLE
Proxy SHA update

### PR DESCRIPTION
**Pull request by rshriram**
Bug fix for envoy crashes that @vadimeisenbergibm reported in Envoy repo (https://github.com/envoyproxy/envoy/issues/1693)

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -39,7 +39,7 @@ bind(
 git_repository(
     name = "envoy",
     remote = "https://github.com/envoyproxy/envoy.git",
-    commit = "ec3446e9b3631568db6223e9eaf37fc6ce9f6cf2", # Sep 17, 2017 (added support for regex route matching)
+    commit = "d49bb3e967c16028520395efa56edf1af96f8a0e", # Sep 20, 2017 (bugfix for envoy crash w/ HTTP2 promise frames)
```
Comment (ldemailly) [inline WORKSPACE]: this is a commit from envoy master 2h ago
it has
https://travis-ci.org/envoyproxy/envoy/builds/277913909?utm_source=github_status&utm_medium=notification
red builds; maybe we need to wait a bit to get it ?

**Comment by ldemailly** [comment]
the sha for the bug fix seems to be https://github.com/envoyproxy/envoy/commit/d3452612b573656e093b322f2f0d8bda31f272db ?

**Comment by rshriram** [comment]
So it seems to have been some merge race.. Waiting for this to complete: https://github.com/envoyproxy/envoy/pull/1706

**Comment by rshriram** [comment]
the tip also includes a coverty scan fix. https://github.com/envoyproxy/envoy/pull/1703

**Comment by rshriram** [comment]
https://travis-ci.org/envoyproxy/envoy/builds/277950151?utm_source=github_status&utm_medium=notification all green

**Comment by ldemailly** [comment]
8 mins of stability :-) sounds like a good bet for istio :-)

**Comment by ldemailly** [comment]
Joke apart, the PR build passed but the master commit one is still running
https://github.com/envoyproxy/envoy/commit/2c7ee712a0795e79732546de8b01e1e7319d809c
let's wait a bit on this (at least that their build passes)

**Comment by rshriram** [comment]
That commit completed. Their status message is not updating. See the travis board here:
https://travis-ci.org/envoyproxy/envoy/builds/277950151?utm_source=github_status&utm_medium=notification

**Pull request merged**